### PR TITLE
Event Assistance

### PR DESCRIPTION
**Pull request by chirica2501**

### CODE REVIEW

File: teckers/teckers/Alert.swift [hunk 1]
```diff
@@ -12,38 +12,35 @@ import UIKit
...
-            alert = UIAlertController(title: self.title, message: self.massage, preferredStyle: UIAlertController.Style.alert)
+        return alert
+    }
+    
+    func showActionSheetAssistance(id: String, success : @escaping (_ response: Int) -> Void) -> UIAlertController {
+        let alert = UIAlertController(title: self.title, message: self.massage, preferredStyle: .actionSheet)
+        
+        alert.addAction(UIAlertAction(title: "Si", style: .default , handler: { (UIAlertAction) in
+            Session.setSessionAssistanceRequest(id: id, success: { response in
+                if response == OK.statusOK.rawValue || response == TypesNetworkErrors.confict.rawValue {
```
Comment (cortestavo) [inline teckers/teckers/Alert.swift]: use a more descriptive name for the enum (OK is not ok)

File: teckers/teckers/Alert.swift [hunk 2]
```diff
@@ -12,38 +12,35 @@ import UIKit
...
+                    success(response)
+                }
+            })
+        }))
+        alert.addAction(UIAlertAction(title: "No🙁", style: .default , handler: { (UIAlertAction) in
```
Comment (cortestavo) [inline teckers/teckers/Alert.swift]: use nil if the completion handler is not needed

File: teckers/teckers/SessionController.swift [hunk 3]
```diff
@@ -159,39 +158,41 @@ extension SessionController: UICollectionViewDataSource {
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         if (collectionView == sessionCollectionView) {
-            let session = collectionView.dequeueReusableCell(withReuseIdentifier: Cell.SessionCell.rawValue, for: indexPath) as! SessionCell
+            let sessionCurrent = collectionView.dequeueReusableCell(withReuseIdentifier: Cell.SessionCell.rawValue, for: indexPath) as! SessionCell
```
Comment (cortestavo) [inline teckers/teckers/SessionController.swift]: should be currentSession

File: teckers/teckers/SessionRouter.swift [hunk 4]
```diff
@@ -11,39 +11,45 @@ import Alamofire
...
         case .getSessions:
             return .get
+        case .setSessionAssistance(_):
+            return .post
         }
+        
     }
-
+    
     var parameters: Parameters? {
```
Comment (cortestavo) [inline teckers/teckers/SessionRouter.swift]: remove parameters

**Review by cortestavo — CHANGES_REQUESTED** [review]

**Review by maggie97 — APPROVED** [review]

**Pull request merged**